h2o-py/tests/testdir_algos/automl/target_encoding/pyunit_automl_gbm_target_encoding.py

```diff
@@ -53,7 +53,7 @@ def titanic_with_te_kfoldstrategy(frame = None, seeds = None):
 
       teColumns = ["home.dest", "cabin", "embarked"]
       targetEncoder = TargetEncoder(x= teColumns, y= targetColumnName,
-                                    fold_column= foldColumnName, blending_avg= True, inflection_point = 3, smoothing = 1)
+                                    fold_column= foldColumnName, blended_avg= True, inflection_point = 3, smoothing = 1)
```

Comment:
Is this to match the R API?  Should this be in a separate PR?

Reply:
@ledell the goal was to match the R API. Ideally it should be in a separate PR, yes. Is it ok to  leave it here?

Reply:
Yeah we can leave it here.  Just wanted to check.  Thanks.